packages/edit-post/src/index.js

```diff
@@ -97,7 +97,9 @@ export function initializeEditor(
 	);
 	registerCoreBlocks();
 	if ( process.env.GUTENBERG_PHASE === 2 ) {
-		__experimentalRegisterExperimentalCoreBlocks( settings );
+		__experimentalRegisterExperimentalCoreBlocks(
+			settings.__unstableEnableFullSiteEditingBlocks
```

Comment:
Good idea 👍 

The only downside is that all those blocks are going to be always sent to the browser in the plugin :( 
Still, it is exactly what we had before so it can be ignored.